2025/08/12 18:47:09 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:47:09 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 451aebb31d03, add metric step
INFO  [alembic.runtime.migration] Running upgrade 451aebb31d03 -> 90e64c465722, migrate user column to tags
INFO  [alembic.runtime.migration] Running upgrade 90e64c465722 -> 181f10493468, allow nulls for metric values
INFO  [alembic.runtime.migration] Running upgrade 181f10493468 -> df50e92ffc5e, Add Experiment Tags Table
INFO  [alembic.runtime.migration] Running upgrade df50e92ffc5e -> 7ac759974ad8, Update run tags with larger limit
INFO  [alembic.runtime.migration] Running upgrade 7ac759974ad8 -> 89d4b8295536, create latest metrics table
INFO  [89d4b8295536_create_latest_metrics_table_py] Migration complete!
INFO  [alembic.runtime.migration] Running upgrade 89d4b8295536 -> 2b4d017a5e9b, add model registry tables to db
INFO  [2b4d017a5e9b_add_model_registry_tables_to_db_py] Adding registered_models and model_versions tables to database.
INFO  [2b4d017a5e9b_add_model_registry_tables_to_db_py] Migration complete!
INFO  [alembic.runtime.migration] Running upgrade 2b4d017a5e9b -> cfd24bdc0731, Update run status constraint with killed
INFO  [alembic.runtime.migration] Running upgrade cfd24bdc0731 -> 0a8213491aaa, drop_duplicate_killed_constraint
INFO  [alembic.runtime.migration] Running upgrade 0a8213491aaa -> 728d730b5ebd, add registered model tags table
INFO  [alembic.runtime.migration] Running upgrade 728d730b5ebd -> 27a6a02d2cf1, add model version tags table
INFO  [alembic.runtime.migration] Running upgrade 27a6a02d2cf1 -> 84291f40a231, add run_link to model_version
INFO  [alembic.runtime.migration] Running upgrade 84291f40a231 -> a8c4a736bde6, allow nulls for run_id
INFO  [alembic.runtime.migration] Running upgrade a8c4a736bde6 -> 39d1c3be5f05, add_is_nan_constraint_for_metrics_tables_if_necessary
INFO  [alembic.runtime.migration] Running upgrade 39d1c3be5f05 -> c48cb773bb87, reset_default_value_for_is_nan_in_metrics_table_for_mysql
INFO  [alembic.runtime.migration] Running upgrade c48cb773bb87 -> bd07f7e963c5, create index on run_uuid
INFO  [alembic.runtime.migration] Running upgrade bd07f7e963c5 -> 0c779009ac13, add deleted_time field to runs table
INFO  [alembic.runtime.migration] Running upgrade 0c779009ac13 -> cc1f77228345, change param value length to 500
INFO  [alembic.runtime.migration] Running upgrade cc1f77228345 -> 97727af70f4d, Add creation_time and last_update_time to experiments table
INFO  [alembic.runtime.migration] Running upgrade 97727af70f4d -> 3500859a5d39, Add Model Aliases table
INFO  [alembic.runtime.migration] Running upgrade 3500859a5d39 -> 7f2a7d5fae7d, add datasets inputs input_tags tables
INFO  [alembic.runtime.migration] Running upgrade 7f2a7d5fae7d -> 2d6e25af4d3e, increase max param val length from 500 to 8000
INFO  [alembic.runtime.migration] Running upgrade 2d6e25af4d3e -> acf3f17fdcc7, add storage location field to model versions
INFO  [alembic.runtime.migration] Running upgrade acf3f17fdcc7 -> 867495a8f9d4, add trace tables
INFO  [alembic.runtime.migration] Running upgrade 867495a8f9d4 -> 5b0e9adcef9c, add cascade deletion to trace tables foreign keys
INFO  [alembic.runtime.migration] Running upgrade 5b0e9adcef9c -> 4465047574b1, increase max dataset schema size
INFO  [alembic.runtime.migration] Running upgrade 4465047574b1 -> f5a4f2784254, increase run tag value limit to 8000
INFO  [alembic.runtime.migration] Running upgrade f5a4f2784254 -> 0584bdc529eb, add cascading deletion to datasets from experiments
INFO  [alembic.runtime.migration] Running upgrade 0584bdc529eb -> 400f98739977, add logged model tables
INFO  [alembic.runtime.migration] Running upgrade 400f98739977 -> 6953534de441, add step to inputs table
INFO  [alembic.runtime.migration] Running upgrade 6953534de441 -> bda7b8c39065, increase_model_version_tag_value_limit
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:47:09 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:47:09 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
[2025-08-12 18:47:09 +0530] [82405] [INFO] Starting gunicorn 23.0.0
[2025-08-12 18:47:09 +0530] [82405] [INFO] Listening at: http://127.0.0.1:5050 (82405)
[2025-08-12 18:47:09 +0530] [82405] [INFO] Using worker: sync
[2025-08-12 18:47:09 +0530] [82406] [INFO] Booting worker with pid: 82406
[2025-08-12 18:47:09 +0530] [82409] [INFO] Booting worker with pid: 82409
[2025-08-12 18:47:09 +0530] [82410] [INFO] Booting worker with pid: 82410
[2025-08-12 18:47:09 +0530] [82411] [INFO] Booting worker with pid: 82411
2025/08/12 18:47:23 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:47:23 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:47:23 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:47:23 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:47:27 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:47:27 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:47:29 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:47:29 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:48:44 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:48:44 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:48:44 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:48:44 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:48:44 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:48:44 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:49:11 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:49:11 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:49:11 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:49:11 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:49:24 INFO mlflow.store.db.utils: Creating initial MLflow database tables...
2025/08/12 18:49:24 INFO mlflow.store.db.utils: Updating database tables
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
2025/08/12 18:50:19 ERROR mlflow.server: Exception on /ajax-api/2.0/mlflow/artifacts/list [GET]
Traceback (most recent call last):
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 594, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 635, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 1093, in _list_artifacts
    response_message = list_artifacts_impl(request_message)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 1115, in list_artifacts_impl
    artifact_entities = _get_artifact_repo(run).list_artifacts(path)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/store/artifact/s3_artifact_repo.py", line 323, in list_artifacts
    for result in results:
                  ^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 272, in __iter__
    response = self._make_request(current_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 360, in _make_request
    return self._method(**current_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 601, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 1074, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (InvalidAccessKeyId) when calling the ListObjectsV2 operation: The AWS Access Key Id you provided does not exist in our records.
2025/08/12 18:50:49 ERROR mlflow.server: Exception on /ajax-api/2.0/mlflow/artifacts/list [GET]
Traceback (most recent call last):
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 594, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 635, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 1093, in _list_artifacts
    response_message = list_artifacts_impl(request_message)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 1115, in list_artifacts_impl
    artifact_entities = _get_artifact_repo(run).list_artifacts(path)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/store/artifact/s3_artifact_repo.py", line 323, in list_artifacts
    for result in results:
                  ^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 272, in __iter__
    response = self._make_request(current_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 360, in _make_request
    return self._method(**current_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 601, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 1074, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (InvalidAccessKeyId) when calling the ListObjectsV2 operation: The AWS Access Key Id you provided does not exist in our records.
2025/08/12 18:51:06 ERROR mlflow.server: Exception on /model-versions/get-artifact [GET]
Traceback (most recent call last):
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/__init__.py", line 83, in serve_model_version_artifact
    return get_model_version_artifact_handler()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 594, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 635, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 2050, in get_model_version_artifact_handler
    return _send_artifact(artifact_repo, artifact_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 582, in _send_artifact
    file_path = os.path.abspath(artifact_repository.download_artifacts(path))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/store/artifact/artifact_repo.py", line 289, in download_artifacts
    if self._is_directory(artifact_path):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/store/artifact/artifact_repo.py", line 188, in _is_directory
    listing = self.list_artifacts(artifact_path)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/store/artifact/s3_artifact_repo.py", line 323, in list_artifacts
    for result in results:
                  ^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 272, in __iter__
    response = self._make_request(current_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 360, in _make_request
    return self._method(**current_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 601, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 1074, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (InvalidAccessKeyId) when calling the ListObjectsV2 operation: The AWS Access Key Id you provided does not exist in our records.
2025/08/12 18:51:16 ERROR mlflow.server: Exception on /ajax-api/2.0/mlflow/artifacts/list [GET]
Traceback (most recent call last):
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 594, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 635, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 1093, in _list_artifacts
    response_message = list_artifacts_impl(request_message)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/server/handlers.py", line 1115, in list_artifacts_impl
    artifact_entities = _get_artifact_repo(run).list_artifacts(path)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/mlflow/store/artifact/s3_artifact_repo.py", line 323, in list_artifacts
    for result in results:
                  ^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 272, in __iter__
    response = self._make_request(current_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/paginate.py", line 360, in _make_request
    return self._method(**current_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 601, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/context.py", line 123, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/amitsahu/Desktop/aion_gen/mlflow/mlflow_POC/.venv/lib/python3.12/site-packages/botocore/client.py", line 1074, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (InvalidAccessKeyId) when calling the ListObjectsV2 operation: The AWS Access Key Id you provided does not exist in our records.
